packages/terra-date-picker/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Fixed
+  * Fixed description id's in DatePicker for better accessibility.
+
```

Comment:
nit: This can be moved to the existing "Fixed" section.

Reply:
updated here https://github.com/cerner/terra-framework/pull/1252/commits/0278a696952f80ca6aca91743ccdc6531926f9cf